.gitignore

```diff
@@ -77,6 +77,7 @@ target/
 .DS_Store
 .sass-cache/
 *.map
+!edivorce/apps/core/static/css/main.css.map
```

Comment:
There is a change in Django 4 where it throws an error if the map file does not exist